core/model/include/Geometry.h

```diff
@@ -117,12 +117,24 @@ namespace lightdb {
     };
     class SpatiotemporalRange: public internal::Range<SpatiotemporalRange, number, internal::limits::spatial>     {
         using internal::Range<SpatiotemporalRange, number, internal::limits::spatial>::Range;
+    public:
+        constexpr SpatiotemporalRange(const double start, const double end)
```

Comment:
Why is this overload needed?  There is a version in the base class and an implicit conversion from double to number.